packages/repl/src/lib/Repl.svelte

```diff
@@ -50,9 +51,18 @@
 
 		injectedCSS = data.css || '';
 
-		await sleep(50);
```

Comment:
Here we don't need to sleep anymore...this was to allow the editor to get it's state before setting the editor state map but since we are loading every file with the new state it's not needed anymore